src/desktop/apps/article/components/layouts/Article.js

```diff
@@ -38,6 +40,42 @@ export default class ArticleLayout extends React.Component {
         article: new ArticleModel(article),
       })
     }
+
+    const editorialAuthDismissedCookie = Cookies.get(
+      'editorial-signup-dismissed'
+    )
+
+    if (!this.props.isLoggedIn && !editorialAuthDismissedCookie) {
+      this.showAuthModal()
+    }
+
+    // setup listener for dismissal cookie
+    mediator.on('modal:closed', this.dismissAuthModal)
+  }
+
+  dismissAuthModal() {
+    Cookies.set('editorial-signup-dismissed', 1, { expires: 864000 })
+  }
+
+  showAuthModal() {
+    if (!this.props.isLoggedIn && !this.props.isMobile) {
+      window.addEventListener(
```

Comment:
Maybe the event listener should be removed on unmount?

Reply:
Yeah I think that makes sense. The listener gets removed after it's fired once. With that in mind do you still think it should be removed after being unmounted?

Reply:
Ooh didn't see the removal of the scroll event, as long as it is happening somewhere should be good.